### DISCUSSION TITLE
Tilted layers

### DISCUSSION THREAD

**Discussion by hentsd** [gempy-project/gempy#580] [Q&A] [answered]
Hey,
What's the easiest way to plot straight tilted layers?
I tried modifying points and orientations but the layers started to wave/curl.

hentsd

**Comment by Leguark** [comment, marked as answer]
Hi @hentsd 
probably the easiest way is to place two surface points in the same location (two points per surface is the minimum the algorithm takes) and one orientation nearby. With that configuration the layers should follow quite well the direction of the orientation
